kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -99,23 +103,19 @@ func (g harness) TestRepo(repo string) (Result, error) {
 	fromRepo, err := g.filenamesFromRepo(repo)
 	if err != nil {
 		log.Printf("Failed to read repo from remote: %v", err)
-		return Result{false, false, 0.0}, nil
+		return Result{false, false, len(fromRepo), 0, 0.0}, nil
```

Comment:
is len(fromRepo) ever going to be nonzero here?

Reply:
Probably not, but this future-proofs it in case we ever start returning partial errors that we can recover from.